draft-ietf-sframe-enc.md

```diff
@@ -1150,12 +1266,18 @@ As mentioned in {{replay}}, senders MUST reject requests to encrypt multiple tim
 with the same key and nonce.
 
 It is not mandatory to implement anti-replay on the receiver side. Receivers MAY
-apply time or counter based anti-replay mitigations.  For example, {{Section
+apply time- or counter-based anti-replay mitigations.  For example, {{Section
 3.3.2 of ?RFC3711}} specifies a counter-based anti-replay mitigation, which
 could be adapted to use with SFrame, using the CTR field as the counter.
 
 ## Metadata
 
+<!--[rfced] Section 9.4. It is unclear what "pure" means in the following sentence. Is the metadata only specified by the application? Please help us clarify.
+
+Original:
+   The metadata input to SFrame operations is pure application-specified
+   data. -->
+
```

Comment:
Yes, it is whatever the application provides, not modified or processed by the SFrame logic except to feed it into the AAD.  Proposed rewording.

```suggestion
```